metrics/validate_drp.yaml

```diff
@@ -0,0 +1,223 @@
+# validate_drp metric definitions
+# Definitions from LPM-17 (the Science Requirements Document).
+# http://ls.st/lpm-17
+
+PA1:
+  description: >
+    The maximum rms of the unresolved source magnitude distribution around the
+    mean value.
+  unit: mmag
+  reference:
+    doc: LPM-17
+    url: http://ls.st/lpm-17
+    page: 21
+
+PF1_minimum_gri:
+  description: >
+    The maximum fraction of magnitudes deviating by more than PA2_minimum_gri
+    from mean (%).
+  unit: ''
```

Comment:
Instead of empty string for unitless quantities, we should use `dimensionless_unscaled`.